src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -881,6 +896,23 @@ private static String processPageSourceForGodMode(String content) {
                 // jQuery-ui CDN
                 .replace("https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js", "${lib.path}/jquery-ui.min.js");
     }
+    
+    private static String processPageSourceForExpectedHtmlRegeneration(String content) {
+        return replaceInjectedValuesWithPlaceholders(content);
+    }
```

Comment:
This method looks redundant but in the spirit of maintainability I think it's better this way?